components/annot-continuous-values.vue

```diff
@@ -53,10 +53,9 @@
 
             ...mapGetters([
 
-                "getPreviewValues",
```

Comment:
What's the reason for removing `getPreviewValues`?

Reply:
So this is an oversight on my part actually. `getPreviewValues` was kind of an imagined getter placed in the initial refactor of this file. At the moment it's necessary because it's called by the `validationItems` function in `annot-continuous-values`, but this will be removed in my next PR for #336 .